pkg/plugins/network/tasks.go

```diff
@@ -105,7 +106,7 @@ type DeployNetworkPlugin struct {
 
 func (d *DeployNetworkPlugin) Execute(runtime connector.Runtime) error {
 	if _, err := runtime.GetRunner().SudoCmd(
-		"/usr/local/bin/kubectl apply -f /etc/kubernetes/network-plugin.yaml --force", true); err != nil {
+		"sudo /usr/local/bin/kubectl apply -f /etc/kubernetes/network-plugin.yaml --force", true); err != nil {
```

Comment:
`runtime.GetRunner().SudoCmd()` will add `sudo` prefix to the cmd string automaticlly.